backend/service/auth/auth_service.go

```diff
@@ -40,7 +40,7 @@ const (
 
 // Service defines the Status service
 type Service struct {
-	GithubBaseURL string
+	GithubBaseURL *url.URL
```

Comment:
@dissoupov just curious any particular reason this to be exportable? I see it is only used in this package and set only in Factory method. Also surprised that lint does not tag it since it is exported without a comment.

Reply:
```
renaming "GithubBaseURL" to "githubBaseURL" would make it unexported breaking references from packages such as "github.com/ekspand/trusty/backend/service/auth_test"
```

Reply:
the comment is only required on exported `struct` not on members (optional)